src/functionalTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/UpdateCaseTest.java

```diff
@@ -21,8 +21,26 @@
 
     @Test
     public void should_update_case() throws JSONException {
```

Comment:
Because we've got two scenarios now, maybe we should add something to the name of this method, making it clear that this is for the manual case update.

Reply:
agree, will rename